app/views/themes/halcyonic/layouts/halcyonic.html.erb

```diff
@@ -1,11 +1,11 @@
-<!DOCTYPE HTML>
+<!DOCTYPE html>
 <!--
 	This set of templates is heavily based on Halcyonic by HTML5 UP:
 	https://html5up.net/halcyonic | html5up.net | @ajlkn
 	Free for personal and commercial use under the CCA 3.0 license:
   html5up.net/license
 -->
-<% if controller_path == 'pages' && @page == Page.default_page %>
+<% if controller_name == 'pages' && @page == Page.default_page %>
```

Comment:
Consider replacing the `@viewing_homepage` instance variable with a helper method in one of `app/helpers/*.rb`, e.g.

```rb
def viewing_homepage?
  controller_name == 'pages' && @page == Page.default_page
end
```

In template:

```erb
<% if viewing_homepage? %>
  ...
<% end %>
```

Reply:
Rubocop yelled at me for using the @page instance variable in a helper, so I tidied it up with a method in the Pages model instead... what do you think? https://github.com/denny/ShinyCMS-ruby/commit/e1e907512887395fea0e09927e94c701798dd221

Reply:
Yep looks good 👍 

I don't think the following is needed here but it can be handy in similar situations (it'd also satisfy Rubocop). 

View helper methods can also be defined in controllers, using `helper_method`: https://api.rubyonrails.org/classes/AbstractController/Helpers/ClassMethods.html#method-i-helper_method

Here, we could define a `current_page` method on `PagesController` and make it available to the view:

```rb
class PagesController ...

  # Makes methods available to views
  helper_method :current_page, :viewing_homepage?

  def current_page
    @page
  end

  def viewing_homepage?
    current_page.default_page?
  end
```